test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerIntegrationTest.cs

```diff
@@ -84,5 +84,21 @@ public Task Build_SimpleMvc_WithServer_UsingDesktopMSBuild_CanBuildSuccessfully(
             Assert.FileExists(result, IntermediateOutputPath, "Whitespace in name.RazorCoreGenerate.cache");
             Assert.FileExists(result, RazorIntermediateOutputPath, "Views", "Home", "Index.cs");
         }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public async Task Build_ErrorInServer_DisplaysErrorInMsBuildOutput()
+        {
+            var result = await DotnetMSBuild(
+                "Build",
+                "/p:_RazorForceBuildServer=true /p:RazorLangVersion=5.0");
+
+            Assert.BuildFailed(result);
+            Assert.BuildOutputContainsLine(result, "5.0 is not a valid language version.");
```

Comment:
What's the full line look like coming from MSBuild?

Reply:
This is the entire line.

Reply:
Is this still the entire line?

Reply:
Plus it probably needs to say `Razor` somewhere in

Reply:
> Is this still the entire line?

Yes, but there is also another line after this that says `rzc generate exited with code -1`.

> Plus it probably needs to say Razor somewhere in

👍